htmlmaker_postprocessing.js

```diff
@@ -95,6 +106,83 @@ fs.readFile(file, function editContent (err, contents) {
     $(this).after(match[4]);
   });
 
+  // remove Section-Blank-Page sections
+  var sectionBlankPage = $("section.blankpage");
+  sectionBlankPage.remove();
```

Comment:
Is there a reason you created this as a variable before removing? If not, simpler is probably better: $("section.blankpage").remove();